.zenodo.json

```diff
@@ -0,0 +1,208 @@
+    },
+    {
+      "name": "Martin Raspaud",
+      "affiliation": "SMHI"
+    },
+    {
+      "name": "Randy Horne",
+      "affiliation": "Excalibur Laboratories Inc"
+    },
+    {
+      "name": "Timothy Whiteaker",
+      "affiliation": "University of Texas"
+    },
+    {
+      "name": "David Blodgett",
+      "affiliation": "USGS"
+    },
+    {
+      "name": "Charlie Zender",
+      "affiliation": "UC Irvine"
+    },
+    {
+      "name": "Daniel Lee",
+      "affiliation": "EUMETSAT"
+    },
+    {
+      "name": "David Hassell",
+      "affiliation": "NCAS and University of Reading",
+      "orcid": "0000-0001-5106-7502"
+    },
+    {
+      "name": "Alan D. Snow",
+      "affiliation": "Corteva Agriscience"
+    },
+    {
+      "name": "Tobias Kölling",
+      "affiliation": "Max Planck Institute for Meteorology"
+    },
+    {
+      "name": "Dave Allured",
+      "affiliation": "NOAA/CIRES/University of Colorado"
+    },
+    {
+      "name": "Aleksandar Jelenak",
+      "affiliation": "HDF Group"
+    },
+    {
+      "name": "Anders Meier Soerensen",
+      "affiliation": "EUMETSAT"
+    },
+    {
+      "name": "Lucile Gaultier",
+      "affiliation": "OceanDataLab"
+    },
+    {
+      "name": "Sylvain Herlédan",
+      "affiliation": "OceanDataLab"
+    },
+    {
+      "name": "Fernando Manzano",
+      "affiliation": "Puertos del Estado, Madrid"
+    },
+    {
+      "name": "Lars Bärring",
+      "affiliation": "SMHI Rossby Centre, Swedish Meteorological and Hydrological Institute",
+      "orcid": "0000-0001-7280-2502"
+    },
+    {
+      "name": "Christopher Barker",
+      "affiliation": "NOAA"
+    },
+    {
+      "name": "Sadie L. Bartholomew",
+      "affiliation": "National Centre for Atmospheric Science and University of Reading",
+      "orcid": "0000-0002-6180-3603"
+    }
+  ],
+  "contributors": [
+    {
+      "type": "ProjectMember",
+      "name": "Guilherme Castelão",
+      "affiliation": "Scripps Institution of Oceanography, UC San Diego",
+      "orcid": "0000-0002-6765-0708"
+    },
+    {"name": "Ryan Abernathey", "affiliation": "Columbia University"},
+    {"name": "Ben Adams", "affiliation": "RPS group"},
+    {"name": "Abel Aoun", "affiliation": "EUMETSAT"},
+    {"name": "Sean Arms", "affiliation": "SpectraLogic"},
+    {"name": "Balaji", "affiliation": "NOAA and Princeton University"},
+    {"name": "Andrew Barna", "affiliation": "SIO/UCSD"},
+    {"name": "Abby Benson", "affiliation": "USGS"},
+    {"name": "Jim Biard", "affiliation": "NOAA"},
+    {"name": "Leon Bilton", "affiliation": "ANU"},
+    {"name": "Benno Blumenthal", "affiliation": "Columbia University"},
+    {"name": "Philip Cameron-Smith", "affiliation": "LLNL"},
+    {"name": "John Caron", "affiliation": "Unidata"},
+    {"name": "Antonio Cofi\u00f1o", "affiliation": "University of Cantabria"},
+    {"name": "Lorenzo Corgnati", "affiliation": "CNR-ISMAR"},
+    {"name": "Ethan Davis", "affiliation": "Unidata"},
```

Comment:
```suggestion
    {"name": "Ethan Davis", "affiliation": "NSF Unidata", "orcid": "0000-0002-8530-2213"},
```

Reply:
Should we systematically include orcid i.d.?  If so, mine is 0000-0002-6491-2135.
Karl